documentation/staging/content/faq/newbie.md

```diff
@@ -0,0 +1,124 @@
+---
+title: "Answers for newcomers"
+date: 2019-09-19T10:41:32-05:00
+draft: false
+weight: 1
+description: "Answers to commonly asked newcomer questions."
+---
+#### What is the WebLogic Server Kubernetes Operator, how can I get started with it, where is its documentation?
+
+It's all [here](https://oracle.github.io/weblogic-kubernetes-operator/).
```

Comment:
This (and any similar URLs) can and should be converted to relrefs.

Reply:
Done. All the others are done also.

Reply:
I don't see the the change. Did you push it?

Reply:
Not yet, a few more to go (waiting for the edit traffic to slow down; I think it's coming).